### PR TITLE
Fix naming inconsistency, moved “hand-holding” notes to comments, misc edit

### PR DESCRIPTION
**Pull request by tomodachi94**
# Naming inconsistency
`crash_report.md` was `crash-report.md`, which was inconsistent with the other files because other files used underscores (_) as a separator, while `crash-report.md` used a dash (-) as a separator.

# Hand Holding Notes to comments
Moved the guidelines to `<!-- comments -->` to prevent clutter in the issues created.

# Miscellaneous edit
Changed “I” to “the developers” as more people may contribute with time.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -7,18 +7,19 @@ assignees: ''
...
+<!--- THIS IS A COMMENT. IT WILL NOT SHOW UP IN THE FINAL ISSUE. DO NOT REMOVE THEM. -->
+
 **Describe the feature you would like to see**
-A clear and concise description of what you would like to see in the mod.
+<!--- A clear and concise description of what you would like to see in the mod.
 PLEASE DO NOT SUBMIT A FEATURE REQUEST FOR THE FOLLOWING:
 - Implement the unused equipment. Yes, they will be added eventually.
-- Guide book. I'm trying to figure out a solution that doesn't require a dependency.
+- Guide book. The developers are trying to figure out a solution that doesn't require a dependency.
 - New boss <insert structure, mob, or loot>. Don't worry, I do have plans.
```
Comment (Andromander) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Missed an identifier here.

Comment (tomodachi94) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Whoops.

Comment (Andromander) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Still missing the identifier here.

**Comment by tomodachi94** [comment]
I don't believe this is a priority for the project or myself, so I am closing this for now. I will revisit this when I have more time.

**Pull request closed without merging**